src/AutoRest.CSharp/Common/Output/Builders/SerializationBuilder.cs

```diff
@@ -617,5 +618,159 @@ private static PropertyBag<ObjectTypeProperty> PopulatePropertyBag(SerializableO
                     new CSharpType(typeof(Dictionary<,>), additionalPropertiesProperty.Declaration.Type.Arguments),
                     shouldExcludeInWireSerialization);
         }
+        public MultipartObjectSerialization BuildMultipartObjectSerialization(ObjectSchema objectSchema, SchemaObjectType objectType)
```

Comment:
do we really need to have this for swagger models?
I suggest to keep limit your change in DPG at first as your first step.
Considering the complexity and consolidation work, we should not change the schema models.

Reply:
keep it this time.